airbyte-integrations/connectors/source-twilio/source_twilio/spec.json

```diff
@@ -0,0 +1,33 @@
+{
+  "documentationUrl": "https://hub.docker.com/r/airbyte/source-twilio",
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Twilio Spec",
+    "type": "object",
+    "required": ["account_sid", "auth_token", "start_date"],
```

Comment:
should this be `api_key` instead of `auth_token`?

Reply:
Nope, but the below value should be `auth_token` instead of `api_key`.